src/app/components/content/partner/partner.component.ts

```diff
@@ -40,11 +43,13 @@ export class PartnerComponent implements OnInit {
     user.emailAddress = this.loginForm.controls.email.value;
     user.password = this.loginForm.controls.password.value;
 
-    this.userService.login(user).subscribe((loggedInUser: User) => {
+    this.authenticateService.login(user).subscribe((loggedInUser: User) => {
         // if good
         this.showInvalidCredentials = false;
+        this.authenticateService.currentUser = loggedInUser;
 
         alert(`Welcome back ${loggedInUser.name}`);
```

Comment:
Don't think the alert is still necessary, is it?

Reply:
Yeah I will removed when you are finished